src/SdnDiagnostics.psd1

```diff
@@ -82,7 +82,7 @@
         'Get-VfpPortLayer',
         'Get-VfpPortRule',
         'Get-VMNetworkAdapterPortProfile',
-        'Install-SdnDiagnostic',
+        'Install-SdnDiagnostics',
```

Comment:
Since Public function changed, Please also update test module to use new name
https://github.com/microsoft/SdnDiagnostics/blob/main/tests/online/wave1/Utilities.Tests.ps1
This will ensure the test covered